lib/cfme_client/bin/rest_api.rb

```diff
@@ -173,7 +174,9 @@ def prompt_get_data
   msg_exit("Script file #{api_script} does not exist") unless File.exist?(api_script)
 else
   api_params = Trollop.options do
-    api_parameters.each { |param| opt param.intern, param, :default => "" }
+    norm_options  = {:default => ""}
+    multi_options = {:default => "", :multi => true}
+    api_parameters.each { |p| opt p.intern, p, (multi_params.include?(p) ? multi_options.dup : norm_options.dup) }
```

Comment:
Prefer .to_sym over .intern (this can be done separately from this PR as this existed before).